CHANGELOG.rst

```diff
@@ -5,11 +5,11 @@ Changelog
 `Unreleased <https://github.com/hydrologie/xhydro_lstm>`_ (latest)
 ------------------------------------------------------------------
 
-Contributors:
+Contributors: Richard Arsenault (:user:`richardarsenault`)
 
 Changes
 ^^^^^^^
-* No change.
+* First functional version of the package. (:pull:`14`)
```

Comment:
You can probably place this under v.0.1.0. We haven't released it yet.

Would you want to park this on PyPI today?

Reply:
I am not sure if we want to put it on PyPI now, but I don't think we will be adding much to this package from now on so... probably?